.github/workflows/ci.yml

```diff
@@ -43,6 +43,32 @@ jobs:
       - name: Run tests
         run: sh ci/run.sh ${{ matrix.os }}
 
+  create_issue:
+    runs-on: ubuntu-latest
+    # the combination of these two lines seems to do the "if build_and_test failed, run this one"
+    needs: build_and_test
+    if: always() && github.event_name == 'schedule' && needs.build_and_test.result == 'failure'
+
+    steps:
+      - name: Create an issue
+        shell: bash
+        run: |
+          readarray -t exists < <(gh issue list --state open --label nightly-CI-failed --json number --jq '.[].number')
+          if [ -n "${exists[@]}" ]; then
+            echo "Not opening another issue, already have ${#exists[@]} issue(s), latest at https://github.com/rust-lang/rust-semverver/issues/${exists[0]}."
+          else
+            cat << EOF | gh issue create \
+              --label nightly-CI-failed \
```

Comment:
the `nightly-CI-failed` label doesn't exist yet for this repo, could you create that @JohnTitor? also do we want any other labels here? (`help wanted`, `good first issue` or something?)

Reply:
Created: https://github.com/rust-lang/rust-semverver/labels/nightly-CI-failed
I think only adding that label is enough for now, at least it isn't always "good first issue".

Reply:
kk :-)